src/js/controller/settings/exportimage/ImageExportController.js

```diff
@@ -17,6 +17,11 @@
     this.addEventListener(scalingFactorInput, 'input', this.onScalingFactorChange_);
     this.updateScalingFactorText_(scalingFactorInput.value);
 
+    var framesPerRowsInput = document.querySelector('.export-frames-per-rows-field');
+    framesPerRowsInput.value = pskl.UserSettings.get(pskl.UserSettings.FRAMES_PER_ROWS);
+    this.addEventListener(framesPerRowsInput, 'change', this.onFramesPerRowsChange_);
+    this.addEventListener(framesPerRowsInput, 'input', this.onFramesPerRowsChange_);
+
```

Comment:
I'm trying to think if there is a good reason to keep this in ImageExportController rather than in PngExportController ? Looking at the code I see we have the scaling factor logic in ImageExport as well, but I think this might be a leftover.

If you tried and it didn't work for some reason, just tell me what was the issue, so that we can file a follow up issue.